packages/amplify-provider-awscloudformation/src/iterative-deployment/deployment-manager.ts

```diff
@@ -93,9 +91,13 @@ export class DeploymentManager {
       assert(cred.region);
       return new DeploymentManager(cred, cred.region, deploymentBucket, eventMap, options);
     } catch (e) {
-      throw new AmplifyError('DeploymentError', {
-        message: 'Could not load configuration',
-      }, e);
+      throw new AmplifyError(
+        'DeploymentError',
+        {
+          message: 'Could not load configuration',
+        },
+        e,
```

Comment:
This seems like an awfully verbose auto format. Anything we can do to shorten this a bit such as no newline before / after the braces?

Reply:
I don't disagree, but also, not really much to do to change this 😅. Prettier's configuration options are (deliberately) limited.
https://prettier.io/docs/en/options.html